### PR TITLE
Add new category for Ethical hacking, web dev and more tools

### PR DESCRIPTION
**Pull request by scorcism**
- > Created new category for Ethical Hacking
- Added imp websites
- Created new category for Web dev
- Added webites
- Added more webstes in tools category

### CODE REVIEW

**Comment by scorcism** [comment]
Hello,
I'm new to this, I don't know why this gave error.

**Comment by JasonDsouza212** [comment]
can you add some screenshots

**Comment by scorcism** [comment]
Yes, of data or error ?
error: https://imgur.com/Wo0PSDk.png

File: src/components/card.js [hunk 1]
```diff
@@ -67,6 +67,22 @@ function Card() {
               title="Useful Tools"
             ></button>
           </div>
+          <div class="button-wrapper">
```
Comment (JasonDsouza212) [inline src/components/card.js:70]: Can you add a screenshot of the website after these changes are applied so that I can see how the UI looks now

**Comment by JasonDsouza212** [comment]
Nothing to worry about merging If I approve your code it will get merged

**Comment by scorcism** [comment]
I have added the SS.

**Comment by JasonDsouza212** [comment]
I'm talking about how the end result looks after you made changes to the Free-Hit website. I need those screenshots

**Comment by scorcism** [comment]
Of entire webiste ?

**Comment by JasonDsouza212** [comment]
nop just the front page

**Comment by scorcism** [comment]
I have created two categories which i have added the screenshots  (https://imgur.com/HvtV66N.png) or you need screenshots of new created cards ?

**Comment by scorcism** [comment]
okay adding the ss in somwtime.

**Comment by JasonDsouza212** [comment]
Okay, I'll add it now, but we should change the UI design because the categories are taking up too much space

**Review by JasonDsouza212 — APPROVED** [review]
Everything looks fine

**Comment by scorcism** [comment]
Yes, ig Issue  #7 is related to the UI chnage

**Pull request merged**